src/styles/pages/_pagination.scss

```diff
@@ -0,0 +1,46 @@
+@import '../settings/colors';
+
+$W_PAGINATION_LINK_SPACING: 6px;
+
+@mixin w-pagination__links($COLOR, $ACTIVE: false) {
+  display: flex;
+  font-size: 20px;
+  justify-content: center;
+  min-width: 20px;
+  padding: 0px $W_PAGINATION_LINK_SPACING;
+
+  a {
```

Comment:
We try to avoid styling generic selectors because it can lead to specificity battles later. I was thinking you could probably get rid of the outer `div` and make this anchor `.w-pagination__link`. The outer div doesn't seem to be adding much.